src/App.jsx

```diff
@@ -1,9 +1,21 @@
 import "./App.css";
+import ListItem from "./components/ListItem";
 
 function App() {
   return (
     <div className="App">
-      <h1>react introductory exercises</h1>
+      <ul>
+        {[
+          "🐶 dog",
+          "🐱 cat",
+          "🐔 chicken",
+          "🐮 cow",
+          "🐏 sheep",
+          "🐴 horse",
+        ].map((item, index) => (
```

Comment:
The way you use these data here doesn't look pretty cool, I would suggest storing them in a constant and then accessing them through that constant, and if possible separate the icon with the animal's name